### DISCUSSION TITLE
Auto ID key column - hide it

### DISCUSSION THREAD

**Discussion by npnini** [Budibase/budibase#14697] [Help]
I defined a table with auto_id number column.
When I define an application screen for the table, I want to set the auto_id as hidden. Is there an option to do that ?
if not - I suggest to add the option.

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-8689/auto-id-key-column-hide-it">BUDI-8689 Auto ID key column - hide it</a></p>
